setup.py

```diff
@@ -119,12 +119,12 @@ def get_numpy_include_dirs():
     author='Michael Davidsaver',
     author_email='mdavidsaver@gmail.com',
     license='BSD',
+    license_files=["LICENSE"],
```

Comment:
Any chance that this change is related to the error I now see from GHA runs?

```
   python -m twine check dist/p4p*.whl
  shell: /usr/bin/bash -e {0}
...
Checking dist/p4p-4.2.0-cp36-cp36m-manylinux1_i686.whl: ERROR    InvalidDistribution: Invalid distribution metadata: unrecognized or    
         malformed field 'license-file'    
```

Reply:
Looks like it's [some combination of build versions](https://github.com/pypa/twine/issues/1216#issuecomment-2606531615), testing some fixes

Reply:
I have pushed ab76714533af44131f9991c89800eafa097d2769 to fix the issue with `actions/upload-artifact@v4` name uniqueness.

Reading through this `twine`, it isn't clear to me if this issue with the `packaging` module is with the P4P build, or with `twine check`?  If the later case, then it might be better to pull in a specific version when installing `twine`?

Reply:
Not sure what the correct key is in `pyproject.toml` but we've always used `license.file` rather than `license-file`:
https://github.com/DiamondLightSource/python-copier-template-example/blob/09e811777d1cd7b25e50d15ef0505eae015486ff/pyproject.toml#L17

Reply:
From my understanding this is the formal documentation for pyproject.toml: https://packaging.python.org/en/latest/specifications/pyproject-toml/#license
Interestingly there is no "license.file" attribute listed there, nor in the "how to" guide on the subject: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files

The key wanted here is almost certainly `license-files`.

Reply:
I'm making a mess of this, forgot it's a `setup.py` file...